sdcm/nemesis.py

```diff
@@ -578,9 +578,12 @@ def disrupt_destroy_data_then_repair(self):  # pylint: disable=invalid-name
 
     def disrupt_destroy_data_then_rebuild(self):  # pylint: disable=invalid-name
         self._set_current_disruption('CorruptThenRebuild %s' % self.target_node)
-        self._destroy_data_and_restart_scylla()
-        # try to save the node
-        self.repair_nodetool_rebuild()
+        with DbEventsFilter(db_event=DatabaseLogEvent.DATABASE_ERROR,
+                            line="std::system_error (error system:104, sendmsg: Connection reset by peer)",
+                            node=self.target_node):
+            self._destroy_data_and_restart_scylla()
+            # try to save the node
+            self.repair_nodetool_rebuild()
```

Comment:
probably this `repair` will not need to be inside the CM, as it should not drop connections like restart does.

Reply:
but need to test this

Reply:
> i have in another test same message (as error) during `CorruptThenRebuild`

@shoshan , if are you already doing this, please add to other tests that this might be relevant.

Reply:
Not here, it should be filtered in the default filter for everything.